tests/osproc/tpath.nim

```diff
@@ -0,0 +1,44 @@
+discard """
+  output: ""
```

Comment:
Nit: you can be more direct here and specify ``action: run``, for more info see here: https://github.com/nim-lang/Nim/tree/devel/tests#action